crates/matrix-sdk/src/oidc.rs

```diff
@@ -421,16 +401,11 @@ impl Oidc {
     /// ```
     ///
     /// [`ClientErrorCode`]: matrix_sdk::oidc::types::errors::ClientErrorCode
-    pub async fn authorize_scope_with_authorization_code<F, Fut>(
+    pub async fn url_for_authorize_scope_with_authorization_code(
```

Comment:
That's a bit of a mouthful. This should be enough:

```suggestion
    pub async fn url_for_scope_with_authorization_code(
```